pmlc/dialect/tile/transforms/pad_ranges.cc

```diff
@@ -79,7 +79,7 @@ void PadRangesPass::runOnFunction() {
     }
     // Get strides for output
     auto outStrides = util::computeStrideArray(op.getResultType(), op.sink());
-    if (!outStrides) {
+    if (!outStrides || outStrides->strides.size() == 0) {
```

Comment:
Use `strides.empty()`. I feel like maybe we should make `util::computeStrideArray` consistently return either nothing or an empty array but not both.  @jbruestle ?